tests/unit/src/test/scala/tests/WorkspaceSymbolLspSuite.scala

```diff
@@ -142,12 +142,15 @@ class WorkspaceSymbolLspSuite extends BaseLspSuite("workspace-symbol") {
       _ = server.workspaceSymbol("scala.None")
       option = ".metals/readonly/scala/Option.scala"
       _ <- server.didOpen(option)
-      references <- server.references(option, "object None")
+      references = server.references(option, "object None")
```

Comment:
doesn't references return future?

Reply:
Before my changes it called `references` on MetalsLanguageServer and returned Future. But this method only wrapped `referencesSync` in `CancelTokens` to introduce cancelability. 
We don't need cancelability here so I changed it to use directly `referencesSync` and not jump into async world not necessarily. 
But it is just cosmetic thing, I can revert that to Future if you want.